### PR TITLE
homeWork5.1

### PR DESCRIPTION
**Pull request by khOksana99**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+        this.price = price;
+    }
+
+    Engine getEngine() {
+        return engine;
+    }
+
+    void setEngine(Engine engine) {
+        this.engine = engine;
+    }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Car.java:24]: add access modifiers

File: src/main/java/core/basesyntax/Engine.java [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
 
-public class Engine {
+public class Engine implements Cloneable {
+    private int countOfHorsePower;
+    private double volume;
+    private int countOfValves;
+    private String type;
+    private int yearOfMade;
 
+    Engine(int countOfHorsePower, double volume, int countOfValves, String type, int yearOfMade) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Engine.java:10]: add the access modifier

File: src/main/java/core/basesyntax/Car.java [hunk 3]
```diff
@@ -1,5 +1,63 @@
...
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private int yearOfMade;
+    private String model;
+    private String producer;
+    private int price;
+
+    Car(Engine engine, int yearOfMade, String model, String producer, int price) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Car.java:10]: add the access modifier

Comment (khOksana99) [reply src/main/java/core/basesyntax/Car.java:10]: done

File: src/main/java/core/basesyntax/Car.java (hunk 1 above)
Comment (khOksana99) [reply src/main/java/core/basesyntax/Car.java:24]: done

File: src/main/java/core/basesyntax/Engine.java (hunk 2 above)
Comment (khOksana99) [reply src/main/java/core/basesyntax/Engine.java:10]: done

**Pull request closed without merging**